library/std/src/sys/pal/uefi/helpers.rs

```diff
@@ -500,3 +500,61 @@ pub(crate) fn get_device_path_from_map(map: &Path) -> io::Result<BorrowedDeviceP
 
     Ok(BorrowedDevicePath::new(protocol))
 }
+
+/// Helper for UEFI Protocols which are created and destroyed using
+/// [EFI_SERVICE_BINDING_PROTCOL](https://uefi.org/specs/UEFI/2.11/11_Protocols_UEFI_Driver_Model.html#efi-service-binding-protocol)
+#[allow(dead_code)]
```

Comment:
```suggestion
#[expect(dead_code)]
```
to ensure these attrs are removed once used (also for all the others)